src/components/Confetti/Confetti.stories.js

```diff
@@ -0,0 +1,54 @@
+import { select, withKnobs } from '@storybook/addon-knobs';
+
+import StorybookMobileDeviceSimulator from '../StorybookMobileDeviceSimulator/StorybookMobileDeviceSimulator.vue';
+import { availableDevices } from '../StorybookMobileDeviceSimulator/StorybookMobileDeviceSimulator';
+import Confetti from './Confetti.vue';
+import ScrollWrapper from '../ScrollWrapper/ScrollWrapper.vue';
+import Screen from '../Screen/Screen.vue';
+import Heading from '../Heading/Heading.vue';
+
+const ConfettiStories = {
+  component: Confetti,
+  title: 'Components/Confetti',
+  decorators: [withKnobs],
+};
+
+const defaultExample = () => ({
+  components: {
+    Confetti,
+    Heading,
+    Screen,
+    ScrollWrapper,
+    StorybookMobileDeviceSimulator,
+  },
+  props: {
+    device: {
+      default: select('Simulated Mobile Device', [...availableDevices], availableDevices[0]),
+    },
+  },
+  template: `
+   <div style="margin: 10px 50px 10px 50px;">
+   <h2><strong>ContentItem:</strong>&nbsp;A list item which usually transitions the user to content in another screen.</h2>
+   <hr>
+   <StorybookMobileDeviceSimulator :device="device">
+     <Screen>
+       <ScrollWrapper>
+         <Confetti :confetti-particles="300"/>
```

Comment:
**One more question**: does the DOM-position of this element matter? if not: please move it to the very bottom (on a new line above line: 44, right above `</Screen>`)

Reply:
Similar to how we normally do for our modal dialog components.